scripts/ci_test_etherscan.sh

```diff
@@ -7,27 +7,44 @@ cd "$DIR" || exit 255
 
 solc-select use 0.4.25 --always-install
 
+delay_no_key () {
+    # Perform a small sleep when API key is not available (e.g. on PR CI from external contributor)
+    if [ "$GITHUB_ETHERSCAN" = "" ]; then
+        sleep $(( ( RANDOM % 5 )  + 1 ))s
```

Comment:
Why randomized instead of a uniform 5s sleep?

Reply:
As this runs in parallel for both windows & linux, I didn't want them to run in sync by chance and potentially starve each other of rate limit. Although on a second thought, the runners probably have different IPs so it shouldn't be an issue. I'll switch it to a normal sleep 👍